### PR TITLE
Android user interaction for launching zoom

### PR DESCRIPTION
**Pull request by ppamid**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,8 +12,12 @@
         });
     return vars;
   }
-  window.location.replace("myadvocateaurora://launch_video"+window.location.search); setTimeout(function () {
```
Comment (ppamid) [inline index.html:12]: How should this be reconciled? iOS detects if the page redirects but Android needs to display a button to launch Zoom for a reasonable amount of time. So keep this line but change the time to a couple of seconds?

Comment (ppamid) [reply index.html:12]: @watts08 does the iOS redirect resolution still work if we extend it to more than a few seconds as suggested above?

Comment (powerje) [reply index.html:12]: Another option could be looking at header information to make a best guess as to whether or not this is iOS / Android / something else and behave differently from there?

Comment (watts08) [reply index.html:12]: on IOS's end @powerje  should the user see the button? Or is it ok? And @ppamid  the extending of seconds i didn't see any issues as a result of it.

Comment (powerje) [reply index.html:12]: I don't think it's a big deal if they see the button momentarily

Comment (watts08) [reply index.html:12]: well they will see it and have to interact with it

Comment (ppamid) [reply index.html:12]: Added some code below to restore the autoredirect functionality on iOS only. @watts08 if you want to test this again

**Pull request merged**